wiki/Help_Centre/en.md

```diff
@@ -157,41 +179,52 @@ If we were to allow everybody to make and play on multiple accounts, the ranked
 
-You get to play on the first account you ever registered with us, and that's it!
+If you currently have more than one account, please contact [accounts@ppy.sh](mailto:accounts@ppy.sh) to get things sorted out.
 
-#### Is it possible to start over or have my account's statistics wiped?
+#### Is it possible to start over/have my account's statistics wiped/have my account deleted?
 
 **Not at the moment.**
 
 We are considering extra 'score reset' purchases from the osu!store for the truly hardcore players who wish nothing but perfection in their stats, but as it currently stands, we cannot wipe an account's statistics for you.
 
-This doesn't mean you can make a new account to do the same thing, either! Having multiple accounts is strongly frowned upon in our community and we will disable any such accounts that are brought to our attention.
+This doesn't mean you can make a new account to do the same thing, either! Having multiple accounts is against the rules, and we will disable any such accounts that are brought to our attention.
+
+If you are covered by the European Union, then you have the option to delete your account. You can read more about this on the [Privacy Page](https://osu.ppy.sh/legal/en/Privacy#your-rights-and-control). If you are not covered by the European Union, then we will not delete your account.
 
 #### Can I play on a computer that another osu! user has used before?
 
 **Yes, absolutely. This includes shared PCs like netcafes or PC bangs.**
 
-So long as you are playing on only your own account, you can play osu! anywhere.
+**So long as you are playing on only your own account**, you can play osu! anywhere!
 
 If you are playing from a public place, make sure to keep a close eye on your account security. Shared computers are often prime targets for malicious software.
 
 #### Can I get a list of all maps I've played or downloaded?
 
 **You can see all maps you've played on the website.**
 
-**If you have an active supporter tag**, you can see all maps you have submitted a score on by going in the Beatmap section, then clicking the "Ranked (played)" filter. Alternatively, you can use [this link](https://osu.ppy.sh/p/beatmaplist?m=-1&r=7&g=0&la=0&ra=) to access the filtered page directly.
-
-On the new website, this filter can be found by clicking "more search options" on the [beatmap list page](https://osu.ppy.sh/beatmapsets) then "Played" in the "PLAYED" section. Alternatively, you can use [this link](https://osu.ppy.sh/beatmapsets?played=played) to access the filtered page directly.
+**If you have an active supporter tag**, you can see all maps you have submitted a score on by going to the [beatmap list page](https://osu.ppy.sh/beatmapsets), clicking "more search options", then selecting the "PLAYED" filter. Alternatively, you can use [this link](https://osu.ppy.sh/beatmapsets?played=played) to access the filtered page directly.
```

Comment:
```suggestion
**If you have an active supporter tag**, you can see all maps you have submitted a score on, by going to the [beatmap list page](https://osu.ppy.sh/beatmapsets), clicking "More Search Options", then selecting the "Played" filter. Alternatively, you can use [this link](https://osu.ppy.sh/beatmapsets?played=played) to access the filtered page directly.

Reply:
not sure what is the reason for the ```Played``` to be fully capitalized

Reply:
Because on the website it is fully capitalized ~~Nevermind it's not anymore~~ ,will change. Will also change the case of More Search Features, but I'm pretty sure a comma there is unnecessary.